### ISSUE TITLE
lut.compute_1d_vmodel_skfmm() fatal RuntimeError when using scikit-fmm=2019.1.30

### ISSUE THREAD

**Issue by TomWinder** [QuakeMigrate/QuakeMigrate#22]
A known bug in scikit-fmm ( https://github.com/scikit-fmm/scikit-fmm/issues/18 ) can lead to lut.compute_1d_vmodel_skfmm() breaking with a RuntimeError:

RuntimeError: Negative discriminant in time marcher quadratic.

As of now this issue is reportedly not well understood. The underlying issue exists in the previous release - scikit-fmm=0.0.9 - but was not known; the change that has been made is to add the test function that gives the RuntimeError.

**Comment by twinder36** [comment]
TL;DR: - we should ALWAYS USE scikit-fmm==2019.1.30 ; hopefully the generic issue with compute_1d_vmodel_skfmm() is fixed, but be aware that this error might reoccur.

Testing showed that 0.0.9 did run without throwing an error, but returned traveltimes of 0.0 s where it was not physically possible. Further inspection suggests the numerical instability which led to the error was due to extrapolation of the velocity model beyond its top and bottom to infinity and -infinity. I have changed the code so that it uses the biggest and smallest floats instead, and so far this appears to have fixed the issue. Comparison with 3DNLLOC and 1D_sweep NLLOC LUTs using the same grid and velocity model shows minimal discrepancy.

**Comment by TomWinder** [comment]
https://github.com/scikit-fmm/scikit-fmm/issues/18 is marked as closed - closing this too.
